src/pages/Label/Label.css

```diff
@@ -6,3 +6,7 @@
 .label-wrapper > * {
   width: 100%;
 }
+
+.label-wrapper h4 {
+  text-transform: uppercase;
```

Comment:
If you only want to change the text-transform property, you can do it from the jsx file itself. Something like ```<h4>{tag.toUpperCase()}</h4>```
I feel that would be better, in case you plan to again change/replace the heading tag, it will be easier for you.

Reply:
Yes, right right. I didn't think from that perspective. I'll fix it. Thank you.